### PR TITLE
Fix #14: jedi>=0.10 compatibility

### LINKED ISSUE #14: Python autocomplete does not work with jedi 0.10
jedi API has changed.

### PR DESCRIPTION
**Pull request by parente**
Standing on the shoulders of metakernel giants:
https://github.com/Calysto/metakernel/pull/129/files

(Thanks @blink1073)

### CODE REVIEW

File: spylon_kernel/init_spark_magic.py [hunk 1]
```diff
@@ -48,16 +49,19 @@ def get_completions(self, info):
             return []
 
         text = info['code']
-        interpreter = Interpreter(text, [self.env])
-
         position = (info['line_num'], info['column'])
-        path = UserContext(text, position).get_path_until_cursor()
-        path, dot, like = completion_parts(path)
-        before = text[:len(text) - len(like)]
+        interpreter = jedi.Interpreter(text, [self.env])
```
Comment (ericdill) [inline spylon_kernel/init_spark_magic.py]: if jedi is not available in the environment, isn't this going to raise a Type error? Something along the lines of `None type has no attribute Interpreter`. Perhaps this needs a `if jedi is not None` somewhere in here?

Comment (parente) [inline spylon_kernel/init_spark_magic.py]: There's a check further up in the function

Comment (ericdill) [inline spylon_kernel/init_spark_magic.py]: :man_facepalming:

Comment (blink1073) [inline spylon_kernel/init_spark_magic.py]: https://www.youtube.com/watch?v=rX7wtNOkuHo  😜

**Pull request merged**